Sources/ApolloCodegenLib/Templates/MockObjectTemplate.swift

```diff
@@ -68,6 +69,19 @@ struct MockObjectTemplate: TemplateRenderer {
     """
   }
 
+  private func conflictingFieldNameProperties(_ fields: [TemplateField]) -> TemplateString {
+    """
+    \(fields.map { """
```

Comment:
Nit: might be more performant to compute an intersection with the `TestMockConflictingFieldNames` set and then just map over those? I could be wrong, but I believe that set intersection computation is very fast b/c of hashing algorithms. This is so so minor though, I'm not really worried about making the change.

Reply:
Nothing about `fields` nor the `FieldCollector` is a `Set` unfortunately, I don't think we're going to get much benefit out of going that route here.